publications-2019.bib

```diff
@@ -1,5 +1,222 @@
+  language = {en},
+  title = {Extended finite element method for approximation of singularities in groundwater flow},
+  publisher = {Unpublished},
+  year = {2019}
+}
+
+@article{Deolmi2019,
+  doi = {10.1016/j.amc.2018.11.059},
+  url = {https://doi.org/10.1016/j.amc.2018.11.059},
+  year = {2019},
+  month = may,
+  publisher = {Elsevier {BV}},
+  volume = {348},
+  pages = {234--256},
+  author = {G. Deolmi and S. M\"{u}ller and M. Albers and P.S. Meysonnat and W. Schr\"{o}der},
+  title = {A reduced order model to simulate compressible flows over an actuated riblet surface},
+  journal = {Applied Mathematics and Computation}
+}
+
+@article{Vassaux2019,
+  doi = {10.1002/adts.201800168},
+  url = {https://doi.org/10.1002/adts.201800168},
+  year = {2019},
+  month = feb,
+  publisher = {Wiley},
+  volume = {2},
+  number = {5},
+  pages = {1800168},
+  author = {Maxime Vassaux and Robert C. Sinclair and Robin A. Richardson and James L. Suter and Peter V. Coveney},
+  title = {The Role of Graphene in Enhancing the Material Properties of Thermosetting Polymers},
+  journal = {Advanced Theory and Simulations}
+}
+
+@article{Freddi2019,
+  doi = {10.1016/j.mechrescom.2019.01.009},
+  url = {https://doi.org/10.1016/j.mechrescom.2019.01.009},
+  year = {2019},
+  month = mar,
+  publisher = {Elsevier {BV}},
+  volume = {96},
+  pages = {29--36},
+  author = {Francesco Freddi},
+  title = {Fracture energy in phase field models},
+  journal = {Mechanics Research Communications}
+}
+
+@article{Bzowski2019,
+  doi = {10.1088/1757-899x/627/1/012018},
+  url = {https://doi.org/10.1088/1757-899x/627/1/012018},
+  year = {2019},
+  month = oct,
+  publisher = {{IOP} Publishing},
+  volume = {627},
+  pages = {012018},
+  author = {Krzysztof Bzowski and Lukasz Rauch and Maciej Pietrzyk},
+  title = {Selection of the optimal heat treatment conditions using advanced phase transformation models},
+  journal = {{IOP} Conference Series: Materials Science and Engineering}
+}
+
+@article{https://doi.org/10.11588/heidok.00027466,
+  doi = {10.11588/HEIDOK.00027466},
+  url = {http://archiv.ub.uni-heidelberg.de/volltextserver/id/eprint/27466},
+  author = {Ponce Bobadilla,  Ana Victoria},
+  keywords = {500 Natural sciences and mathematics,  510 Mathematics},
+  title = {Mathematical Models of Cell Migration and Proliferation in Scratch Assays},
+  publisher = {Heidelberg University Library},
+  year = {2019}
+}
+
+@article{Basak2019,
+  doi = {10.1016/j.cma.2018.08.006},
+  url = {https://doi.org/10.1016/j.cma.2018.08.006},
+  year = {2019},
+  month = jan,
+  publisher = {Elsevier {BV}},
+  volume = {343},
+  pages = {368--406},
+  author = {Anup Basak and Valery I. Levitas},
+  title = {Finite element procedure and simulations for a multiphase phase field approach to martensitic phase transformations at large strains and with interfacial stresses},
+  journal = {Computer Methods in Applied Mechanics and Engineering}
+}
+
+@misc{https://doi.org/10.25673/13842,
```

Comment:
This is a bachelor's thesis. Is there a way how to flag other than `misc`?

Reply:
>How can an dissertation have two authors?!

Ana Victoria Ponce-Bobadilla: https://victoriapb.github.io/

>This is a bachelor's thesis. Is there a way how to flag other than misc?

I don't know. @tjhei

Reply:
See https://github.com/dealii/publication-list#commonly-used-entry-templates